com.unity.visualeffectgraph/Editor/Models/VFXGraph.cs

```diff
@@ -252,6 +252,8 @@ public static bool HasVFXExtension(string filePath)
         {
             if (string.IsNullOrEmpty(filePath))
```

Comment:
Isn't this test redundant then? From doc: "This method also returns false if path is null, an invalid path, or a zero-length string"

Reply:
Yes, good point.
I'll remove it and add more unit tests.